base/server/python/pki/server/__init__.py

```diff
@@ -73,6 +75,25 @@ def instances(cls):
 
         return instances
 
+    @staticmethod
+    def split_cert_id(cert_id):
```

Comment:
In the future, moving a method like this can be done as a separate patch which can be merged without review since it's a simple refactoring operation, but make sure all references to the method are updated in the same patch.